pages/osx/opendiff.md

```diff
@@ -0,0 +1,13 @@
+# opendiff
+
+> Graphically compare files or directories.
+> Built-in command of OS X.
+> More information: <https://www.manpagez.com/man/1/opendiff/>.
+
+- Compare two files:
+
+`opendiff {{path/to/file1}} {{path/to/file2}}`
+
+- Compare two directories:
+
+`opendiff {{path/to/dir1/}} {{path/to/dir2/}}`
```

Comment:
```suggestion
`opendiff {{path/to/directrory1}} {{path/to/directory2}}`
```
Would this be better?

Reply:
Warning: this change may conflict with #9578 PR. Please check it before suggesting PR which fixes placeholders.